### ISSUE TITLE
Ctrl-Z (undo) does not scroll the view to the location in question

### ISSUE THREAD

**Issue by quartertone** [standardnotes/forum#427]
When using the Undo (ctrl-Z) feature, the edit is undone, but the display does not scroll/jump to the location of the "undo" as would be expected (as is the standard behavior in most text editors). This is noticeable if the note is long enough to need a scrollbar, and if the edit is outside of the currently displayed section.

**Comment by mobitar** [comment]
Which editor is this with?

**Comment by quartertone** [comment]
In my testing, I found this behavior in:
Plain editor
LaTeX editor
Plus editor
Simple markdown editor
Simple task editor

The cursor doesn't jump to the undo location for the above editors.

It worked as expected for Code editor, Minimal Markdown editor, and Adv markdown editor.

Also, I happened to notice that the Vim editor doesn't have a scrollbar either. I'm guessing (per the appearance) that it shares some code, and therefore bugs, with the Code editor.

cheers!
